### PR TITLE
Cumulus 1176 files-to-granules

### PR DESCRIPTION
**Pull request by mennovandiermen**
**Summary:** Summary of changes

Addresses [CUMULUS-1176: Create separate task for getCmrFiles and file reassociation logic currently in PostToCmr.](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1176)

## Changes

* Adds new `files-to-granules` task to handle converting array-of-files to granule object.
* Integration tests with new task will be included in `move-granules` refactor PR.

## PR Checklist

- [X] Update CHANGELOG
- [X] Unit tests
- [X] Adhoc testing

### CODE REVIEW

**Comment by mennovandiermen** [comment]
Would like some feedback on ideas for more tests and whether this should be merged into a feature branch instead so that integration tests are available.

File: tasks/files-to-granules/schemas/output.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            "items": {
+              "type": "object",
+              "required": ["name"],
+              "properties": {
+                "name": {
+                  "type": "string"
+                },
+                "filename": {
+                  "type": "string"
+                },
```
Comment (Jkovarik) [inline tasks/files-to-granules/schemas/output.json]: This task needs to take into account the addition of the fileType to the Granule->Files schema

File: tasks/files-to-granules/schemas/output.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          "files": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": ["name"],
+              "properties": {
+                "name": {
+                  "type": "string"
+                },
+                "filename": {
```
Comment (Jkovarik) [inline tasks/files-to-granules/schemas/output.json]: I believe this needs fileType as well

Comment (mennovandiermen) [reply tasks/files-to-granules/schemas/output.json]: @Jkovarik 2f75072

File: tasks/files-to-granules/index.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+ *
+ * @returns {Object} - Granules object
+ */
+function filesToGranules(event) {
+  return convertFileURIArrayToGranuleObjectArray(event);
+}
+
+exports.filesToGranules = filesToGranules;
+
+function handler(event, context, callback) {
```
Comment (Jkovarik) [inline tasks/files-to-granules/index.js]: 👍  as a refactor to the current workflow logic.

**Pull request merged**